Code/autopkg

```diff
@@ -735,9 +747,8 @@ Example: '%s repo-add recipes'
 ..adds the autopkg/recipes repo from GitHub."""
         % ("%prog", verb, "%prog")
     )
-
     # Parse arguments
-    arguments = parser.parse_args(argv[2:])[1]
+    arguments = common_parse(argv[2:])[1]
```

Comment:
`common_parse()` requires two arguments.

Reply:
How embarrassing. I tested on master, not on my branch...